node/src/chain_spec/oak.rs

```diff
@@ -13,8 +13,11 @@ use primitives::{AccountId, AuraId, Balance};
 
 const TOKEN_SYMBOL: &str = "OAK";
 const SS_58_FORMAT: u32 = 51;
+const STAGING_SS_58_FORMAT: u32 = 52;
 static RELAY_CHAIN: &str = "rococo-local";
+static STAGING_RELAY_CHAIN: &str = "rococo";
 const REGISTERED_PARA_ID: u32 = 2114;
+const REGISTERED_STAGING_PARA_ID: u32 = 2115;
```

Comment:
this paraID is open, but i think we need santiago to claim it before we can actually use